content/blog/2023/2023-10-18-intro-to-ktor.md

```diff
@@ -0,0 +1,257 @@
+---
+title: "Introduction to Ktor"
+categories: ["Kotlin"]
+date: 2023-10-18 00:00:00 +1100 
+authors: [ezra]
+excerpt: "In this tutorial, we'll discuss the Ktor framework"
+image: images/stock/0104-on-off-1200x628-branded.jpg
+url: introduction-to-ktor
+---
+
+In today's fast-paced world, web application development is a critical domain for businesses and developers. Building web applications that are efficient, scalable, and easy to maintain is a challenging task. Enter Ktor, a powerful, asynchronous, and lightweight framework for building web applications and APIs using the Kotlin programming language. Ktor offers a modern approach to web development that has gained significant popularity in recent years.
```

Comment:
"In today's fast-paced world" is a very lame intro to an article. Very generic. :)